lib/convection/control/cloud.rb

```diff
@@ -114,6 +114,18 @@ def diff(to_stack, options = {}, &block)
           return
         end
 
+        errors = []
+        stacks.each_value do |stack|
+          if stack.error?
+            errors << stack.errors.collect { |x| x.exception.message }
```

Comment:
When errors happen in converges, we pass back an array of `Error` objects.  For diff errors, we pass back an array of error messages.  Why this inconsistency?\